src/service/impl/graphql-client.ts

```diff
@@ -0,0 +1,26 @@
+import ApolloClient, { DefaultOptions } from 'apollo-client';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { HttpLink } from 'apollo-link-http';
+import fetch from 'cross-fetch';
+
+const link = new HttpLink({
+  uri: 'https://api.entur.io/journey-planner/v2/graphql',
+  fetch
+});
+
+const defaultOptions: DefaultOptions = {
+  watchQuery: {
+    fetchPolicy: 'cache-and-network',
+    errorPolicy: 'ignore'
+  },
+  query: {
+    fetchPolicy: 'network-only',
```

Comment:
Always fetch new data, but also populate the cache. This is used to manually diff the cache on realtime.